components/badge/metadata/metadata.json

```diff
@@ -177,5 +177,5 @@
   ],
   "system-theme": [],
   "passthroughs": [],
-  "high-contrast": []
+  "high-contrast": ["--highcontrast-spectrum-badge-border-color"]
```

Comment:
I'm not sure if it makes a difference in this case to edit the file directly, or if you need to re-run locally to have this updated to remove `spectrum`?

Reply:
Good catch! In this case it was the only change, but I re-ran `yarn-ci` to verify. ✨